src/Pact/Compile.hs

```diff
@@ -142,7 +142,16 @@ doMetaBody exp  = case exp of
   body              -> Just (pure Nothing, body)
 
 doModule :: [Exp] -> Info -> Info -> Compile (Term Name)
-doModule (EAtom n Nothing Nothing _:ESymbol k _:es) li ai =
+doModule (EAtom n Nothing Nothing _ : ELiteral (LString k) _ : es) li ai =
+  handleModule n k es li ai 
+
```

Comment:
Please be consistent with other code and do not introduce blank lines between pattern matches